app/views/layouts/_user_config.html.erb

```diff
@@ -39,10 +39,10 @@
       } else {
         document.body.classList.add("user-tags-followed-max");
       }
-      sidebarTags.style.height = "100vh"
-      sidebarTags.style.backgroundColor = "green"
     }
   } catch(e) {
+    setTimeout(function(){
```

Comment:
This is definitely a workaround, so I'll open up an issue about sorting this out.